content/_partners/faq._en.md

```diff
@@ -108,7 +108,7 @@ logistics_accordion:
         id: b-8
         title: Can Login.gov authenticate or proof non-U.S. citizens or non-U.S. immigrants (not a U.S. citizen, U.S. national, lawful permanent resident, or traveling to the United States on an immigrant visa)?
         content: >-
-            <a href="https://login.gov/help/manage-your-account/international-phone-support/" class="external-link">Click here for a complete list of international phone numbers that Login.gov supports for authenticating end-users </a>. In addition to a phone number, at this time Login.gov requires users to proof with a state-issued ID card and social security number.
+            Check <a href="https://login.gov/help/manage-your-account/international-phone-support/" class="external-link">International phone number support</a> for a complete list that Login.gov supports for authenticating end-users. In addition to a phone number, at this time Login.gov requires users to proof with a state-issued ID card and social security number.
```

Comment:
Even though it's styled like an external link, I would make this an internal link (and switch to markdown so it gets the [base_url.rb](https://github.com/18F/identity-site/blob/main/_plugins/base_url.rb) plugin to auto-add the prefix so that preview URLs stay on the preview page)
```suggestion
            Check [International phone number support](/help/manage-your-account/international-phone-support/){:class="external-link"} for a complete list that Login.gov supports for authenticating end-users. In addition to a phone number, at this time Login.gov requires users to proof with a state-issued ID card and social security number.
```